ecl/hqlcpp/hqlsource.cpp

```diff
@@ -4432,10 +4426,10 @@ void MonitorExtractor::buildKeySegmentExpr(BuildMonitorState & buildState, KeySe
                     {
                         createStringSet(subctx, targetSet, curSize, selectorInfo.selector->queryType());
                         OwnedHqlExpr targetVar = createVariable(targetSet, makeVoidType());
-                        callAddAll(subctx, targetVar);
+                        callAddAll(subctx, targetVar); // Not sure we need this really - library translates to wild anyway
```

Comment:
I think needed because if it is used in combination with other conditional keyed filters